tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I've merged them up. Resolving the Angular typings from `node_modules` only works with a top level `tsconfig.json`.